telnyx_rtc/build.gradle

```diff
@@ -102,6 +102,8 @@ android {
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles 'proguard-rules.txt'
+
+        buildConfigField "java.util.concurrent.atomic.AtomicBoolean", "IS_TESTING", "new java.util.concurrent.atomic.AtomicBoolean(false)"
```

Comment:
Maybe for another iteration, we can build a Wrapper Singleton of BugSnag and inject it as well, that checks this variable, so we don't have to if-else every time we use it.. don't know how many places we will need it though